google-cloud-pubsub/lib/google/cloud/pubsub/subscription.rb

```diff
@@ -456,10 +459,13 @@ def dead_letter_max_delivery_attempts
         #
         # This field will be honored on a best effort basis. If this parameter is 0, a default value of 5 is used.
         #
-        # The dead letter topic must also be set. See {#dead_letter_topic=} and {#dead_letter_topic}.
+        # The dead letter topic must also be set. See {#dead_letter_topic=}, {#dead_letter_topic},
```

Comment:
Clarify: "The dead letter topic must be set *first*"

Reply:
Thanks, updated.